lib/supavisor/db_handler.ex

```diff
@@ -53,23 +53,25 @@ defmodule Supavisor.DbHandler do
     socket_opts = [
       :binary,
       {:packet, :raw},
-      {:active, true},
+      {:active, false},
```

Comment:
Start off in not-active mode. Allows us to pull data out of socket for SSL (vs writing a bunch of receivers for responses). Mirrors what Postgrex does.